### PR TITLE
Add dracula theme style

### PR DESCRIPTION
**Pull request by fbuys**
Adding it to the demo website: [Here](https://github.com/elixir-makeup/makeup_demo/pull/14) and [here](https://github.com/elixir-makeup/makeup_demo/pull/15).

See: https://draculatheme.com/ 
See: https://spec.draculatheme.com/#sec-Prelude

Here is a screenshot of the applied styles from the demo website:
image

### CODE REVIEW

**Comment by josevalim** [comment]
@tmbb i assume we are happy with adding new themes?

thanks @fbuys !

**Comment by tmbb** [comment]
@josevalim Yes, I have no problem with new themes

File: lib/makeup/styles/html/style_map.ex [hunk 1]
```diff
@@ -345,6 +345,34 @@ defmodule Makeup.Styles.HTML.StyleMap do
...
+      :operator => "#FF92DF",
+      :punctuation => "#F8F8F2",
+      :string => "#F1FA8C",
+      :string_symbol => "#FFB86C",
+    }
+  )
+
+  @doc """
+    The *dracula* style. Example [here](https://elixir-makeup.github.io/makeup_demo/elixir.html#dracula).
+    """
```
Comment (josevalim) [inline lib/makeup/styles/html/style_map.ex]:
```suggestion
  The *dracula* style. Example [here](https://elixir-makeup.github.io/makeup_demo/elixir.html#dracula).
  """
```

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**